### DISCUSSION TITLE
No Favicon - just the Joomla one

### DISCUSSION THREAD

**Discussion by krippels** [templaza/astroid-framework#449] [Q&A]
I have set a favicon in astriod called favicon.png with 96x96 px, but in the browser I only see the joomla-logo. What am I doing wrong? Is there anything else to do? Thanks for your help!

**Comment by Chacapamac** [comment]
First, after inserting your icon in Astroid settings, be sure to clear all Astroid/joomla/Browser caches. Reload, It should work. 

**If not Working, try that...**
_“It is weird because I was expecting that Astroid Settings will automatically override the “by default” Joomla icon?”_
Look for the Joomla default icon “favicon.ico” in  yoursite/media/system/images/favicon.ico. Keep a copy and trash it. 

Now, enter your icon Image in Astroid  Administration and clear all Astroid/joomla/Browser caches. Reload, It should work.

**Comment by krippels** [comment]
Thanks for your quick help - it worked!

**Reply by Chacapamac** [reply]
@krippels  Have a good Day my friend, in it’s day an age we all need good day!
